druid/src/contexts.rs

```diff
@@ -71,8 +69,8 @@ pub struct LifeCycleCtx<'a> {
 /// in the widget's appearance, to schedule a repaint.
 ///
 /// [`request_paint`]: #method.request_paint
-pub struct UpdateCtx<'a, 'b: 'a> {
-    pub(crate) text_factory: &'a mut Text<'b>,
+pub struct UpdateCtx<'a> {
+    //pub(crate) text_factory: &'a mut Text<'b>,
```

Comment:
Probably this should be removed, it sounds like all access is now through the `WinHandle`.